.gitignore

```diff
@@ -6,8 +6,11 @@ docs/build/
 docs/site/
 benchmark/.results/*
 benchmark/.tune.jld
+benchmark/Manifest.toml
+.benchmarkci
 *.cov
 /Manifest.toml
+notes
```

Comment:
this is specific to your setup, no need for it in Graphs.jl

Reply:
right